packages/app/src/systems/SignUp/components/CreatePassword/CreatePassword.tsx

```diff
@@ -120,6 +120,7 @@ export function CreatePassword({
                   autoComplete="new-password"
                   placeholder="Confirm your password"
                   aria-label="Confirm Password"
+                  onPaste={(e) => e.preventDefault()}
```

Comment:
thanks for your contribution

this behavior is helpful for tests/development although I agree with the concern 
can you do a conditional using `IS_DEVELOPMENT` to only prevent when not in development mode ?

Reply:
Change implemented